packages/orchestration/src/typeGuards.js

```diff
@@ -18,3 +20,9 @@ export const Proto3Shape = {
 };
 
 export const CoinShape = { value: M.bigint(), denom: M.string() };
```

Comment:
In the `AnyJson` version of `Coin`, bigints are cast to strings. It seems this can be removed as we are now returning `ChainAmount`. The interface guard for `getBalance` should return `ChainAmountShape`.